packages/driver/cypress/integration/e2e/multi-domain/commands/multi_domain_misc.spec.ts

```diff
@@ -104,3 +73,37 @@ context('multi-domain misc', { experimentalSessionSupport: true, experimentalMul
     })
   })
 })
+
+it('verifies number of cy commands', () => {
+  // @ts-ignore
+  // remove 'getAll' command since it's a custom command we add for our own testing and not an actual cy command
+  const actualCommands = _.reject(Object.keys(cy.commandFns), (command) => command === 'getAll')
```

Comment:
```suggestion
  const actualCommands = Cypress._.reject(Object.keys(cy.commandFns), (command) => command === 'getAll')
```

Can save the need to import lodash by using the instance already attached to Cypress. Can also remove the import above, of course.

Reply:
updated [c3bc2e7](https://github.com/cypress-io/cypress/pull/20467/commits/c3bc2e78ced0bd9f735f1c77c1bbe6915f5af2e2)